Shopify/cli
# Update URLs even without any webs, to enable GraphiQL

### WHY are these changes introduced?

Functions need GraphiQL to accomplish various tasks. But in extension-only apps, we are not updating OAuth callback URLs, so GraphiQL isn't available.



### WHAT is this pull request doing?


Prompts if necessary, and updates URLs regardless of whether there are webs. This way, extension-only apps can still run GraphiQL.

Note that at least one access scope must be defined; otherwise, GraphiQL won't appear.

### How to test your changes?


1. Create an extension-only app.
2. Add a function. (recommendation: product discount)
3. Add an access scope. (recommendation: `write_discounts`)
4. Run `dev`. You should be prompted to approve updating the URLs, and you should be able to open GraphiQL.

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
